lib/vagrant-orchestrate/command/push.rb

```diff
@@ -56,7 +56,8 @@ def execute
           argv = parse_options(opts)
           return unless argv
 
-          guard_clean unless ENV["VAGRANT_ORCHESTRATE_NO_GUARD_CLEAN"]
+          guard_clean unless ENV["VAGRANT_ORCHESTRATE_NO_GUARD_CLEAN"] && \
+                             @env.vagrantfile.config.orchestrate.disable_commit_guard
```

Comment:
Seems like it says run guard clean unless the env variable is set and the config is set. I thought either env variable of this config could disable guard clean.